cmd/apps/opa_gatekeeper.go

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) arkade author(s) 2020. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package apps
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/alexellis/arkade/pkg/apps"
+	"github.com/alexellis/arkade/pkg/k8s"
+	"github.com/alexellis/arkade/pkg/types"
+
+	"github.com/alexellis/arkade/pkg"
+	"github.com/spf13/cobra"
+)
+
+func MakeInstallOPAGateKeeper() *cobra.Command {
+	var opaGatekeeper = &cobra.Command{
+		Use:          "opa-gatekeeper",
+		Short:        "Install OPA Gatekeeper",
+		Long:         `Install Open Policy Agent Gatekeeper`,
+		Example:      `  arkade install opa-gatekeeper`,
+		SilenceUsage: true,
+	}
+
+	opaGatekeeper.Flags().Bool("update-repo", true, "Update the helm repo")
+	opaGatekeeper.Flags().Bool("persistence", false, "Enable persistence")
+	// gatekeeper chart default namespace is fixed in helm chart
+	opaGatekeeper.Flags().StringArray("set", []string{},
+		"Use custom flags or override existing flags \n(example --set persistence.enabled=true)")
+
+	opaGatekeeper.PreRunE = func(command *cobra.Command, args []string) error {
+		_, err := command.Flags().GetString("kubeconfig")
+		if err != nil {
+			return fmt.Errorf("error with --kubeconfig usage: %s", err)
+		}
+
+		_, err = command.Flags().GetBool("wait")
+		if err != nil {
+			return fmt.Errorf("error with --wait usage: %s", err)
+		}
+
+		_, err = command.Flags().GetBool("persistence")
+		if err != nil {
+			return fmt.Errorf("error with --persistence usage: %s", err)
+		}
+
+		_, err = command.Flags().GetBool("update-repo")
+		if err != nil {
+			return fmt.Errorf("error with --update-repo usage: %s", err)
+		}
+
+		_, err = command.Flags().GetStringArray("set")
+		if err != nil {
+			return fmt.Errorf("error with --set usage: %s", err)
+		}
+
+		return nil
+	}
+
+	opaGatekeeper.RunE = func(command *cobra.Command, args []string) error {
+		kubeConfigPath, _ := command.Flags().GetString("kubeconfig")
+		wait, _ := command.Flags().GetBool("wait")
+		updateRepo, _ := command.Flags().GetBool("update-repo")
+		persistence, _ := command.Flags().GetBool("persistence")
+		customFlags, _ := command.Flags().GetStringArray("set")
+
+		fmt.Printf("Using kubeconfig: %s\n", kubeConfigPath)
+		arch := k8s.GetNodeArchitecture()
+		fmt.Printf("Node architecture: %q\n", arch)
+
+		overrides := map[string]string{}
+		overrides["persistence.enabled"] = strings.ToLower(strconv.FormatBool(persistence))
```

Comment:
I think this is a copy-paste error - I can't see persistence as a chart option. (So we dont need the arkade flag or this override)

Reply:
Resolved